src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-unit-form/attachment-unit-form.component.ts

```diff
@@ -88,7 +88,7 @@ export class AttachmentUnitFormComponent implements OnInit, OnChanges {
             name: [undefined as string | undefined, [Validators.required, Validators.maxLength(255)]],
             description: [undefined as string | undefined, [Validators.maxLength(1000)]],
             releaseDate: [undefined as dayjs.Dayjs | undefined],
-            version: [{ value: 1, disabled: true, readOnly: true }],
+            version: [1],
```

Comment:
The field should still be disabled! Also see https://github.com/ls1intum/Artemis/pull/8114/files#r1510001615

```suggestion
            version: [{ value: 1, disabled: true }],
```